### ISSUE TITLE
External USB camera

### ISSUE THREAD

**Issue by maniootek** [HelgeApps/droid_dashcam_faq#15]
I connected usb camera to my Samsung Galaxy S20 FE and other app (USB Camera Viewer) can use this camera. I tried to find this camera under camera list in Droid Dashcam app but can't find in the list. Is it possible to use USB camera with Droid Dashcam app? I wish to use external usb small camera sticked to my windshield and phone hidden in the storage. My phone will be hidden from the sun.

**Comment by HelgeApps** [comment]
Yes, that app uses a specific library to identify usb cameras. I may implement a new similar app for this case which works only with USB cameras because the logic is quite different and there are many annoying USB permissions which users need to grant every time they connect a usb camera to their phones, so it's quite uncomfortable

**Comment by maniootek** [comment]
`I may implement a new similar app for this case`
ok please let me know

**Comment by maniootek** [comment]
Hello. Did you make any progress with USB external camera for Droid DashCam app or other app?

**Comment by EdnilJ** [comment]
I'd be interested in an answer, too. THX

**Comment by c0dev0id** [comment]
Same here. I'm unsure about the permission claim. I installed https://play.google.com/store/apps/details?id=infinitegra.app.usbcamera and it was able to record from my usb camera without requesting anything unusual. Only regular Camera access permissions. (On Android 14)
